env.default

```diff
@@ -4,7 +4,7 @@ PULSE_API_DOMAIN=https://network-pulse-api-production.herokuapp.com
 PULSE_DOMAIN=www.mozillapulse.org
 # Always add a protocol (ex: https://) in front of the NETWORK_SITE_URL, including for localhost
 NETWORK_SITE_URL=https://foundation.mozilla.org
-TARGET_DOMAIN=foundation.mozilla.org
+TARGET_DOMAINS=foundation.mozilla.org
```

Comment:
this also needs a `DOMAIN_REDIRECT_MIDDLEWARE_ENABLED=False`, or django will crash, citing
```
Traceback (most recent call last):
  File ".virtualenvs\foundation.mozilla.org-4aDC7Y6j\lib\site-packages\environ\environ.py", line 273, in get_value
    value = self.ENVIRON[var]
  File ".virtualenvs\foundation.mozilla.org-4aDC7Y6j\lib\os.py", line 678, in __getitem__
    raise KeyError(key) from None
KeyError: 'DOMAIN_REDIRECT_MIDDLEWARE_ENABLED'
```